src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('tbody');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(person => {
+  let gender;
 
-// write your code here
+  if (person.sex === 'm') {
+    gender = 'Male';
+  } else {
+    gender = 'Female';
+  }
+
+  table.insertAdjacentHTML('beforeend',
+    `
+    <tr>
+      <td>${person.name}</td>
+      <td>${gender}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.ceil(person.died / 100)}</td>
```

Comment:
To calculate the century, you should use the 'born' date instead of 'died'. Replace 'person.died' with 'person.born' in line 26.